help/export-your-organization.md

```diff
@@ -9,13 +9,16 @@
 
 Zulip has high quality export tools that can be used to migrate from the
 hosted Zulip Cloud service (zulip.com) to your own servers. There are
-three types of Zulip Cloud exports.
+four types of Zulip Cloud exports.
 
 * **Export of public data**: An export of all users, settings, and all the data that
   appears in public streams.
 * **Full export with member consent**: Everything in export of public data, plus all
   the private data of members that opt-in to the export.
 * **Full export without member consent**: All the data in the organization.
+* **Compliance export**: A targeted, human-readable export of messages
+  matching some criteria (e.g. sender, recipient, message keyword, or
```

Comment:
Can we make it clear that you can use a combination of these criteria? E.g., "some criteria" -> "some combination of criteria".